docs/examples.rst

```diff
@@ -11,10 +11,10 @@ Orbit-by-Orbit Plots
 --------------------
 
 Plotting a series of orbit-by-orbit plots is a great way to become familiar
-with a satellite data set. If the data set doesn't come with orbit information, this can
-be a challenge. Orbits also go past day breaks, so if data comes in daily files
-this requires loading multiple files at once, joining the data together, etc.
-pysat goes through that trouble for you.
+with a satellite data set. If the data set doesn't come with orbit information
+this can be a challenge. Orbits also go past day breaks. If data comes in daily
+files this requires loading multiple files at once, joining the data together,
+etc. pysat goes through that trouble for you.
```

Comment:
```suggestion
this can be a challenge. Orbits also go past the ends of files that are saved at non-orbital cadences (e.g., daily).  Thus, loading an orbit requires loading multiple files at once, joining the data together, and selecting the orbital period. pysat goes through that trouble for you.
```

The line length is now too long, so that will need to be fixed.  Suggestion boxes are pretty bad at line length limits :P

Reply:
The style guide I have been following says the line length is 79 characters to fit a standard terminal.  I have seen guides that also suggest a length of 72 characters but most of the documents were already seemed to be around 79 characters so I went with that.

Reply:
We use 80 characters in the code, but 79 is standard and fine.  What i mean to say here, is that determining the number of characters in the suggestion box is difficult so if you commit this suggestion as is, it will need to be updated afterwards.